cmd/log.go

```diff
@@ -39,12 +38,15 @@ var logCmd = &cobra.Command{
 	Args: func(cmd *cobra.Command, args []string) error {
 		return nil
 	},
+	PreRunE: setPagedCobraOutput,
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := context.Background()
+		var err error
 
-		root, err := index.RootDirectory(".")
+		var root string
+		root, err = index.RootDirectory(".")
 		if err != nil {
-			logrus.Fatal(err)
+			return
```

Comment:
We should at least print out the error somehow, even if we cut execution short. Unfortunately, as noted above `logurs.Fatal` is a bad call.